Documentation/ceph-pool-crd.md

```diff
@@ -106,6 +106,44 @@ external-cluster-console# rbd mirror pool peer bootstrap import <token file path
 
 See the official rbd mirror documentation on [how to add a bootstrap peer](https://docs.ceph.com/docs/master/rbd/rbd-mirroring/#bootstrap-peers).
 
+### Data spread across subdomains
+
+Imagine the following topology with datacenters containing racks and then hosts:
+
+```text
+.
+├── datacenter-1
+│   ├── rack-1
+│   │   ├── host-1
+│   │   ├── host-2
+│   └── rack-2
+│       ├── host-3
+│       ├── host-4
+└── datacenter-2
+    ├── rack-1
+    │   ├── host-1
+    │   ├── host-2
```

Comment:
The hosts are in the same K8s cluster, so the host names still need to be unique across the datacenters.